lib/src/Base/Common/ResourceMap.cxx

```diff
@@ -1317,6 +1317,11 @@ void ResourceMap::loadDefaultConfiguration()
   // NAIS parameters //
   addAsScalar("NAIS-DefaultRhoQuantile", 0.25);
 
+  // Cross Entropy Importance Sampling parameters //
+  addAsScalar("CrossEntropyImportanceSampling-DefaultRhoQuantile", 0.25);
```

Comment:
I dont know if we need to distinguish a default value for the 2 child classes, maybe the one from the parent class is enough